gui/WorldView.cxx

```diff
@@ -979,27 +1018,40 @@ void WorldView::computeContour(double threshold)
 //-----------------------------------------------------------------------------
 bool WorldView::loadMesh(QString const& path)
 {
+  QTE_D();
+
   QFileInfo fileInfo{path};
   if (!fileInfo.exists() || !fileInfo.isFile())
   {
     return false;
   }
 
   QGuiApplication::setOverrideCursor(Qt::WaitCursor);
-
-  // Create the vtk pipeline and read the mesh
-  vtkNew<vtkPLYReader> reader;
-  reader->SetFileName(qPrintable(path));
-  reader->Update();
-  QGuiApplication::restoreOverrideCursor();
-
-  auto mesh = vtkPolyData::SafeDownCast(reader->GetOutput());
-  if (mesh->GetPointData()->GetNumberOfArrays() <= 0)
```

Comment:
Maybe we shouldn't lose this check?

Reply:
(Note: as discussed elsewhere, this wasn't doing exactly what I thought it was doing. We might want to revisit checking that we actually loaded data, though. Not necessarily in this PR.)